Microsoft.Toolkit.Uwp.UI.Controls.DataGrid/DataGrid/DataGridComboBoxColumn.cs

```diff
@@ -0,0 +1,457 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Reflection;
+using Microsoft.Toolkit.Uwp.UI.Controls.DataGridInternals;
+using Windows.UI.Text;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    /// <summary>
+    /// Represents a <see cref="DataGrid"/> column that hosts content that in its cells. When in edit mode data can be changed to a value from a collection hosted in a ComboBox.
```

Comment:
"content that" seems wrong. The text box type uses "textual content".

Reply:
Thank you for spotting that. Will change it.